script_library/pre-flight-checks.sh

```diff
@@ -117,7 +117,11 @@ check_ansible_requirements (){
     # If ara is requested
     if [[ ${USE_ARA:-False} == "True" ]]; then
         check_python_requirement 'ara'
-        python3 -m ara.setup.env > ${SOCOK8S_WORKSPACE_BASEDIR}/${SOCOK8S_ENVNAME}-workspace/ara.rc
+        ARA_PATH=$(python3 -m ara.setup.path)
+        cat << EOF > ${SOCOK8S_WORKSPACE_BASEDIR}/${SOCOK8S_ENVNAME}-workspace/ara.rc
```

Comment:
this sucks :(

Reply:
I will propose a patch to ara I guess :)

Reply:
https://review.opendev.org/#/c/679281/1

Reply:
Is not there need to generate env and output to ara.rc first ? And then append to file content new exports.

I mean this line (# 120) in existing code .
`python3 -m ara.setup.env > ${SOCOK8S_WORKSPACE_BASEDIR}/${SOCOK8S_ENVNAME}-workspace/ara.rc`

Reply:
I will fix the line that's missing about the plugin location if that's what you meant.

Reply:
yes output of ara.setup.path though not sure if you still need that as you have already referred those paths in action and callback plugins env variables export

Reply:
Still not sure what you mean here.